src/Home/NewTopBanner/NewTopBanner.js

```diff
@@ -0,0 +1,183 @@
+      color: 0xe1e1e1,
+      program: function(context) {
+        context.beginPath();
+        context.arc(0, 0, 0.6, 0, PI2, true);
+        context.fill();
+      },
+    });
+
+    var i = 0;
+
+    for (var ix = 0; ix < AMOUNTX; ix++) {
+      for (var iy = 0; iy < AMOUNTY; iy++) {
+        particle = particles[i++] = new Particle(material);
+        particle.position.x = ix * SEPARATION - (AMOUNTX * SEPARATION) / 2;
+        particle.position.z = iy * SEPARATION - (AMOUNTY * SEPARATION) / 2;
+        scene.add(particle);
+      }
+    }
+
+    renderer = new CanvasRenderer();
+    renderer.setSize(window.innerWidth, window.innerHeight);
+    container.appendChild(renderer.domElement);
+
+    document.addEventListener("mousemove", onDocumentMouseMove, false);
+    document.addEventListener("touchstart", onDocumentTouchStart, false);
+    document.addEventListener("touchmove", onDocumentTouchMove, false);
+
+    //
+
+    window.addEventListener("resize", onWindowResize, false);
+    animate();
+  }
+
+  function onWindowResize() {
+    windowHalfX = window.innerWidth / 2;
+    windowHalfY = window.innerHeight / 2;
+
+    camera.aspect = window.innerWidth / window.innerHeight;
+    camera.updateProjectionMatrix();
+
+    renderer.setSize(window.innerWidth, window.innerHeight);
+  }
+
+  //
+
+  function onDocumentMouseMove(event) {
+    mouseX = event.clientX - windowHalfX;
+    mouseY = event.clientY - windowHalfY;
+  }
+
+  function onDocumentTouchStart(event) {
+    if (event.touches.length === 1) {
+      event.preventDefault();
+
+      mouseX = event.touches[0].pageX - windowHalfX;
+      mouseY = event.touches[0].pageY - windowHalfY;
+    }
+  }
+
+  function onDocumentTouchMove(event) {
+    if (event.touches.length === 1) {
+      event.preventDefault();
+
+      mouseX = event.touches[0].pageX - windowHalfX;
+      mouseY = event.touches[0].pageY - windowHalfY;
+    }
+  }
+
+  //
+
+  function animate() {
+    requestAnimationFrame(animate);
+
+    render();
+  }
+
+  function render() {
+    camera.position.x += (mouseX - camera.position.x) * 0.05;
+    camera.position.y += (-mouseY - camera.position.y) * 0.05;
+    camera.lookAt(scene.position);
+
+    var i = 0;
+
+    for (var ix = 0; ix < AMOUNTX; ix++) {
+      for (var iy = 0; iy < AMOUNTY; iy++) {
+        particle = particles[i++];
+        particle.position.y =
+          Math.sin((ix + count) * 0.3) * 50 + Math.sin((iy + count) * 0.5) * 50;
+        particle.scale.x = particle.scale.y =
+          (Math.sin((ix + count) * 0.3) + 1) * 2 +
+          (Math.sin((iy + count) * 0.5) + 1) * 2;
+      }
+    }
+
+    renderer.render(scene, camera);
+
+    count += 0.1;
+  }
+
+  return (
+    <div
+      id="particleThreeJsHoverBase"
+      className={styles.particleThreeJsHoverBase}
+    >
+      <div className={styles.title}>
+        <h1
+          className={styles.threeTitleHead}
+          style={{ fontSize: "80px", color: "white" }}
```

Comment:
If classname is provided avoid inline css unless you have to manipulate that CSS using some states!